### PR TITLE
feat: google login

### PR DESCRIPTION
**Pull request by JakubKostrzewsky**
Feat: login with Google

## Type of Change

- [x] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

File: lib/dependencies.dart [hunk 1]
```diff
@@ -1,8 +1,16 @@
+import 'package:appunite_internal_app/core/dependencies.dart' as core;
 import 'package:appunite_internal_app/features/login/dependencies.dart' as login;
 import 'package:get_it/get_it.dart';
+import 'package:google_sign_in/google_sign_in.dart';
 
 final sl = GetIt.asNewInstance();
 
 void setupDependencies() {
+  final GoogleSignIn googleSignIn = GoogleSignIn(
```
Comment (JakubKostrzewsky) [inline lib/dependencies.dart:9]: This value will not be used in the future so it is safe.

File: android/app/build.gradle [hunk 2]
```diff
@@ -49,8 +49,8 @@ android {
 
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
-        applicationId "com.example.verygoodcore.appunite_internal_app"
-        minSdkVersion 16
+        applicationId "com.appunite.appuniteinternalapp"
```
Comment (mhnowak) [inline android/app/build.gradle:52]: maybe just? `com.appunite.internalapp` or `com.appunite.internal.app`?

File: ios/Runner/Info.plist [hunk 3]
```diff
@@ -25,6 +25,25 @@
 	<string>$(FLUTTER_BUILD_NAME)</string>
 	<key>CFBundleSignature</key>
 	<string>????</string>
```
Comment (mhnowak) [inline ios/Runner/Info.plist:27]: `com.appunite...`

File: ios/Runner/Info.plist [hunk 4]
```diff
@@ -25,6 +25,25 @@
...
+			</array>
+		</dict>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>com.googleusercontent.apps.219486636480-ocgtsubr5fd795ss4jjn1tpgaieorl4o</string>
+			</array>
+		</dict>
```
Comment (mhnowak) [inline ios/Runner/Info.plist:45]: What are these for?

File: lib/dependencies.dart (hunk 1 above)
Comment (mhnowak) [reply lib/dependencies.dart:9]: Imo if we're not gonna use it in the feature we shouldn't commit it. Maybe just add a field in the envs and locally replace it.

File: test/features/login/login_cubit_test.dart [hunk 5]
```diff
@@ -0,0 +1 @@
+
```
Comment (mhnowak) [inline test/features/login/login_cubit_test.dart:1]: Oops..

File: android/app/build.gradle (hunk 2 above)
Comment (JakubKostrzewsky) [reply android/app/build.gradle:52]: I kept the previous naming for appId.

**Pull request merged**